server/api-server/index.js

```diff
@@ -18,23 +18,29 @@ class ApiServer {
   }
 
   _startApi() {
-    this.app = express();
-    this.app.disable('x-powered-by');
-    this.app.use(bodyParser.json());
-    this.app.use(cors());
-    this.app.use('/api', router(this));
-    this.app.use('/images', express.static(path.join(__dirname, '..', 'images')));
-
-    return getDevelopmentCertificate('frontend-grocer', { installCertutil: true }).then((ssl) => {
-      https.createServer(ssl, this.app).listen(this.program.apiPort, () => {
-        chalk.white(` - Starting API on https://localhost:${this.program.apiPort}\n\n`)
+    return new Promise((resolve) => {
+      this.app = express();
+      this.app.disable('x-powered-by');
+      this.app.use(bodyParser.json());
+      this.app.use(cors());
+      this.app.use('/api', router(this));
+      this.app.use('/images', express.static(path.join(__dirname, '..', 'images')));
+      debug('Attempting to get certificate');
```

Comment:
Changes to this file are unrelated. I just added some debug statements so we can better troubleshoot around cert generation if student issues arise.